v2/pkg/utils/yaml/yaml_decode_wrapper.go

```diff
@@ -0,0 +1,35 @@
+package yaml
+
+import (
+	"io"
+	"strings"
+
+	"github.com/go-playground/validator/v10"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+var validate *validator.Validate
+
+// DecodeAndValidate is a wrapper for yaml Decode adding struct validation
+func DecodeAndValidate(r io.Reader, v interface{}) error {
+	if err := yaml.NewDecoder(r).Decode(v); err != nil {
+		return err
+	}
+	if validate == nil {
+		validate = validator.New()
+	}
+
+	if err := validate.Struct(v); err != nil {
+
```

Comment:
is it possible to remove the empty line?

Reply:
sure!